### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by rochdiNassah**

### CODE REVIEW

File: src/MarshalsPsr7RequestsAndResponses.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ protected function toPsr7Response(Response $response): ResponseInterface
      *
      * This instance can turn a PSR-7 request into an HttpFoundation request.
      *
-     * @return \Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory
+     * @return \Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactoryInterface
```
Comment (nunomaduro) [inline src/MarshalsPsr7RequestsAndResponses.php:61]: Maybe is `Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface`?

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: src/MarshalsPsr7RequestsAndResponses.php (hunk 1 above)
Comment (rochdiNassah) [reply src/MarshalsPsr7RequestsAndResponses.php:61]: It is already changed to ```HttpFoundationFactoryInterface```.

Comment (nunomaduro) [reply src/MarshalsPsr7RequestsAndResponses.php:61]: But the FQN is not correct. Check this: https://github.com/symfony/psr-http-message-bridge/blob/main/HttpFoundationFactoryInterface.php.

Comment (rochdiNassah) [reply src/MarshalsPsr7RequestsAndResponses.php:61]: I checked and it seems correct as the last change.

Comment (Namoshek) [reply src/MarshalsPsr7RequestsAndResponses.php:61]: No, it is not correct. The `HttpFoundationFactoryInterface` is imported as `Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface` in the same file on line 14:

https://github.com/laravel/octane/blob/dc393ebf101c880d4b96c5560d2d50bea6adb91a/src/MarshalsPsr7RequestsAndResponses.php#L14

Comment (rochdiNassah) [reply src/MarshalsPsr7RequestsAndResponses.php]: Changed, yes you're right.
thanks!

**Pull request merged**